### DISCUSSION TITLE
Grid Image Swapping - Angular

### DISCUSSION THREAD

**Discussion by NirantharikaSelvachandran** [community/community#83899] [Programming Help]
### Select Topic Area

Question

### Body

I require your Angular expertise! 🚀 I'm currently working on implementing a grid image-swapping feature in our Angular application, and I've encountered a challenge that's proving to be quite a puzzle. As a believer in the power of community collaboration, I'm reaching out to seek your guidance!

🚧 The Challenge:
My goal is to enable users to seamlessly swipe through images in a grid layout, both vertically and horizontally. The backend is set up to handle the data, and the frontend, built with Angular, stores images as an array. However, I'm facing difficulties when it comes to implementing the swapping functionality in both directions. 🤷‍♂️

🙏 How You Can Help:
1. If you've successfully implemented grid image swiping in an Angular application, especially in both vertical and horizontal directions, I'd love to learn from your experience.
2. Any Angular-specific insights, tips, or recommendations on overcoming challenges related to this feature would be immensely appreciated.
3. Open to a virtual chat or a review session? I'm eager to connect with Angular enthusiasts!

Your Angular wisdom is invaluable, and I believe that with your support, I can crack the code for this grid image-swapping challenge in our Angular app!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @NirantharikaSelvachandran ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
